lib/validators/commit.js

```diff
@@ -73,14 +77,22 @@ class Commit extends Validator {
       orderedCommits = [orderedCommits[orderedCommits.length - 1]]
     }
 
-    const commitMessages = orderedCommits.map(commit => commit.message)
+    let commitMessages = orderedCommits.map(commit => commit.message)
+    let FAIL_MESSAGE = DEFAULT_FAIL_MESSAGE
```

Comment:
Since this isn't a const, I suggest changing this to `failMessage` as the variable name.